curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -711,16 +711,19 @@ std::vector<ObjectChunkInfo> FileCacheManager::GetReadChunks(
                     VLOG(9)
                         << "chunkTmp offset:" << chunkTmp.s3ChunkInfo.offset()
                         << ", len:" << chunkTmp.s3ChunkInfo.len()
-                        << ", objectoffset:" << chunkTmp.objectOffset;
+                        << ", objectoffset:" << chunkTmp.objectOffset
+                        << ",j:"  << j;
```

Comment:
These are not your codes? remove them pls.

Reply:
> These are not your codes? remove them pls.

fix